src/parsers/forecast_parser.jl

```diff
@@ -140,34 +170,51 @@ end
 function add_forecast_data!(
                             infos::ForecastInfos,
                             simulation::AbstractString,
-                            category::Type{<:Component},
-                            component_name::Union{AbstractString, Nothing},
+                            component::Component,
                             label::AbstractString,
+                            per_unit::Bool,
                             data_file::AbstractString,
                            )
-    if !haskey(infos.data_files, data_file)
-        if !isfile(data_file)
-            msg = "Timeseries file doesn't exist : $file_path"
-            throw(DataFormatError(msg))
-        end
+    timeseries = _add_forecast_data!(infos, data_file, get_name(component))
 
-        infos.data_files[data_file] = read_time_array(data_file, component_name)
+    normalize = component isa LoadZones ? true : false
```

Comment:
This is not good.  I'm not sure how to provide this customization.  I don't think it should go into timeseries_pointers.csv, as that describes the raw data, not what we should do with it.  There is no LoadZones config parameter because there is no LoadZones raw data.  Any opinion @claytonpbarrows

Reply:
Per discussion with Clayton I pulled out the normalization code.  It is now up to the user to supply timeseries data that is already normalized if that is what they want.